app/models/user.rb

```diff
@@ -37,4 +41,19 @@ def find_badge_assignment(badge)
   def admin_module?(current_module)
     roles.pluck(:name).include? "admin_#{current_module}"
   end
+
+  def self.order_ids_by_votes(votes, order)
+    if order == :desc
+      votes.sort_by{|_k,v| v}.reverse.to_h.keys
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.